.gitignore

```diff
@@ -1 +1,2 @@
+text.md
 
```

Comment:
[OPTIONAL] Kindly consider adding `node_modules` to the `.gitignore` file as well because you might accidentally create the `node_modules` directory when installing lintels locally.